app/models/miq_request_workflow/dialog_field_validation.rb

```diff
@@ -8,7 +8,11 @@ def validate_tags(field, values, _dlg, fld, _value)
     required_tags = fld[:required_tags].to_miq_a.collect(&:to_sym)
     missing_tags = required_tags - selected_tags_categories
     missing_categories_names = missing_tags.collect do |category|
-      Classification.find_by_name(category.to_s).description rescue nil
+      begin
+        Classification.lookup_by_name(category.to_s).description
+      rescue StandordError
```

Comment:
typo
```suggestion
      rescue StandardError
```

Reply:
Sure, let me try to split it into more commits. Currently, I just 1. first make alias and deprecation warning; 2. change the callers. Do you think I could make each commit for each file, e.g. only methods in tag.rb and related ones?

Thanks for catching this typo! This is scary since it is a bug hard to find. Let me try to add unit test for it.

The deprecation warning will help us identify the existing callers and prevent users from using the find_by_*.  The annoying part is that we might get tons of deprecation warning for other repo calling these find_by_*,

And I am checking the other repos. Right now, the one has most deprecation warnings is automation engine. Others have just a few.

I have made some drafts for other major repos https://github.com/ManageIQ/manageiq-automation_engine/pull/373 https://github.com/ManageIQ/manageiq-ui-classic/pull/6219 https://github.com/ManageIQ/manageiq-api/pull/675

Reply:
How about removing the deprecation in this PR and add the deprecation after merging most of other repo's change?